modules/puppet_connect/plans/test_input_data.pp

```diff
@@ -13,6 +13,19 @@
 #
 plan puppet_connect::test_input_data(TargetSpec $targets = 'all') {
   $targs = get_targets($targets)
+  $unique_plugins = $targs.group_by |$t| {$t.plugin_hooks['puppet_library']}
+  if ($unique_plugins.keys.length > 1) {
+    out::message('Multiple puppet_library plugin hooks detected')
+    $unique_plugins.each |$plug, $target_list| {
+      $target_message = if ($target_list.length > 10) {
+                          $target_list.length
```

Comment:
I think this means to condense the list?

Reply:
I think it should be `"${target_list.length} targets"` and remove the word `targets` from the message below. Otherwise it will be "Plugin hook [...] configured for [foo.example.com, bar.example.com] targets" which is a weird one.